docs/gettingstarted.rst

```diff
@@ -0,0 +1,55 @@
+..
+    Copyright 2017-2020 - Swiss Data Science Center (SDSC)
+    A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+    Eidgenössische Technische Hochschule Zürich (ETHZ).
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+Getting Started
+===============
+.. _gettingstarted-reference:
+
+Interaction with the platform can take place via the command-line
+interface (CLI).
+
+Start by creating for folder where you want to keep your Renku project:
```

Comment:
This can be simplified with

```
renku init my-renku-project
cd my-renku-project
```

I think it was written before `renku init` could take a directory as an argument...